meta/conf/machine/salvator-x-m3.conf

```diff
@@ -0,0 +1,3 @@
+require xt-distro-machine.inc
+
+MACHINEOVERRIDES =. "rcar:rcar-gen3:"
```

Comment:
I'm just curious why did you miss r8a7796 SoC name here? Did you just discard it as meaningless?